lib/tasks/create_printers.rake

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+namespace :printers do
+  task create: :environment do
+    [
+      { name: 'g216bc', labware_type: 'tube' },
+      { name: 'h105bc', labware_type: 'tube' },
+      { name: 'ssrtubebc-sq1', labware_type: 'tube' },
+      { name: 'aa309bc1', labware_type: 'tube' },
+      { name: 'g311bc1', labware_type: 'tube' },
+      { name: 'aa309bc3', labware_type: 'tube' }
+    ].each do |options|
+      Printer.create_with(options).find_or_create_by!(name: options[:name])
```

Comment:
Do we need to add `create_with` here? I think `find_or_create_by!(options)` would do what `create_with` is trying to do?

Reply:
This is what we are doing elsewhere. Happy to leave it as is.